node/src/main/scala/coop/rchain/node/web/WebApiRoutes.scala

```diff
@@ -70,9 +74,12 @@ object WebApiRoutes {
         handleRequest[Unit, B](req, _ => f)
     }
 
-    // TODO: Create generic encoders/decoders for
-    // ADT's with discriminator field
+    implicit val encodeBongInfo: Encoder[BondInfo] = deriveEncoder[BondInfo]
+    implicit val encodeByteString: Encoder[ByteString] =
+      Encoder.encodeString.contramap[ByteString](b => PrettyPrinter.buildStringNoLimit(b))
```

Comment:
You can supply function directly without creating another lambda.

Reply:
Do you mean
```
Encoder.encodeString.contramap[ByteString](PrettyPrinter.buildStringNoLimit)
```?